From 2c91cf9870e147f1d2b0e5f4149fa4aa999818e3 Mon Sep 17 00:00:00 2001
From: Marc Kleine-Budde <mkl@pengutronix.de>
Date: Sat, 9 Jan 2010 19:22:37 +0100
Subject: [PATCH 1/2] don't include LDFLAGS int pkg-config libs

curl saves the LDFLAGS set during configure into its pkg-config file.
So -Wl stuff ends up in the .pc file, which is really bad, and breaks
if there are multiple -Wl in our LDFLAGS (which are in PTXdist)

This bug was submitted to the curl bug tracking system:
https://sourceforge.net/tracker/?func=detail&aid=2893592&group_id=976&atid=100976

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>

20091106: rsc: added to bug tracker
20091117: rsc: applied in upstream! \o/
---
 libcurl.pc.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libcurl.pc.in b/libcurl.pc.in
index 25beadd..78ea89c 100644
--- a/libcurl.pc.in
+++ b/libcurl.pc.in
@@ -35,6 +35,6 @@ Name: libcurl
 URL: http://curl.haxx.se/
 Description: Library to transfer files with ftp, http, etc.
 Version: @VERSION@
-Libs: -L${libdir} -lcurl @LDFLAGS@ @LIBS@
+Libs: -L${libdir} -lcurl @LIBS@
 Libs.private: @LIBCURL_LIBS@ @LIBS@
 Cflags: -I${includedir}
-- 
1.6.6

